app/releases/Releases.tsx

```diff
@@ -0,0 +1,168 @@
+import { ReleasesResponse } from "@/lib/types";
+import { Repository } from "@/lib/types";
+import { Release } from "@/lib/types";
+import Link from "next/link";
+import Markdown from "@/components/Markdown";
+import { FiExternalLink, FiGithub } from "react-icons/fi";
+import LoadingText from "@/components/LoadingText";
+
+export default async function Releases(props: { className?: string }) {
+  const accessToken = process.env.GITHUB_PAT;
```

Comment:
Use `env` from typesafe envs. This was recently updated in https://github.com/coronasafe/leaderboard/commit/63f4bb5079a163c305fe0d730a5cacf76a332d03